src/aat/resources/json/decision/standard_decision_reply.json

```diff
@@ -0,0 +1,5 @@
+{
+  "decision_reply": "decision_accepted",
+  "decision_reply_reason": "some reason",
+  "author_reference_id": "some author id"
```

Comment:
author_reference_id is actually the IDAM reference passed in the header.  It shouldn't been in the payload.

In the meantime, I'm trying to find out from Alec, what IDAM fields are actually passed in the header